### ISSUE TITLE
[feature] Compound interest calculator

### ISSUE THREAD

**Issue by 1-eric** [sadellie/unitto#149]
It would be nice to have a compound interest calculator that can calculate the absolute and percentage gain after a period of time.

There are two situations:
1) Where money is deposited only once (at the beginning);
2) Where money is deposited at the beginning and every next month/year.

The amount of each deposit and the interest rate is always the same.
